dev-libs/libconfig/libconfig-1.5.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="A simple C/C++ configuration file library"
+DESCRIPTION="Libconfig is a simple library for processing structured \
+configuration files. Libconfig is very compact, a fraction of the size \
+of the expat XML parser library. This makes it well-suited for \
+memory constrained systems like handheld devices."
+HOMEPAGE="http://www.hyperrealm.com/libconfig/"
+COPYRIGHT="2005-2015 Mark A Lindner"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="http://www.hyperrealm.com/libconfig/libconfig-$portVersion.tar.gz"
+CHECKSUM_SHA256="e31daa390d8e4461c8830512fe2e13ba1a3d6a02a2305a02429eec61e68703f6"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
@return, sorry for this late reply. I tried to build on x86_gcc2 but got some errors, among which these 3:
```
cc1: Invalid option `-Wextra'
cc1: Invalid option `-Wdeclaration-after-statement'
cc1: Invalid option `-Wno-unused-parameter'
```
Since it does not seem to build with on x86_gcc2 primary arch, could you please add secondary arch support to this recipe?
To do this, you need to change ARCHITECTURES and add SECONDARY_ARCHITECTURES to have:
```
ARCHITECTURES="!x86_gcc2 x86 x86_64"
SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```
Hmm, maybe what we need is just:
```
ARCHITECTURES="x86 x86_64"
SECONDARY_ARCHITECTURES="x86"
```
@korli, I don't remember what's the rule for a package that only provides libraries. Should we keep the "!x86_gcc2" or drop it?
Thanks!